docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,120 @@
+# Provider Networks
+
+
+## Table of Contents
+- [Provider Networks](#provider-networks)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Common prerequisites](#common-prerequisites)
+  - [Considerations when creating provider networks](#considerations-when-creating-provider-networks)
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in a data center.
+Example of network types in this category include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
```

Comment:
```suggestion
Network types in this category include flat (untagged), VLAN (802.1Q tagged), and VXLAN. 
```

Can we be clearer about what "this category" is? It reads ambiguously to me whether you're saying that flat, VLAN, and VXLAN are provider networks or physical networks in the current construction.